src/plugins/data/public/ui/ui_service.ts

```diff
@@ -83,7 +57,6 @@ export class UiService implements Plugin<IUiSetup, IUiStart> {
       ),
       SearchBar,
       SuggestionsComponent,
```

Comment:
Continue a comment from above, it seems like we do not expose the createEditor function from here. do people access it from the SearchBar?